disdrodb/l0/check_configs.py

```diff
@@ -69,48 +289,77 @@ def check_bin_consistency(sensor_name: str) -> None:
     np.testing.assert_allclose(velocity_bin_upper - velocity_bin_width / 2, velocity_bin_center)
 
 
-def check_variable_keys_consistency(sensor_name: str) -> None:
-    """Check attributes consistency from config file.
+def get_chunksizes(sensor_name: str, file_name: str) -> list:
+    """get chunksizes from config file.
 
     Parameters
     ----------
     sensor_name : str
         Name of the sensor.
+    file_name : str
+        File name (bins_velocity.yml or bins_diameter.yml)
+
+    Returns
+    -------
+    list
+        list of chunksizes (center, bounds, width)
     """
-    description_dict = get_description_dict(sensor_name)
-    units_dict = get_units_dict(sensor_name)
-    long_name_dict = get_long_name_dict(sensor_name)
-    data_format_dict = get_data_format_dict(sensor_name)
-    encoding_dict = get_L0B_encodings_dict(sensor_name)
-    encoding_vars = set(encoding_dict.keys())
-    data_format_vars = set(data_format_dict.keys())
-    long_name_vars = set(long_name_dict.keys())
-    units_vars = set(units_dict.keys())
-    description_vars = set(description_dict.keys())
-
-    encoding_vars.difference(data_format_vars)
-    encoding_vars.difference(units_vars)
-    encoding_vars.difference(description_vars)
-    encoding_vars.difference(long_name_vars)
-
-    data_format_vars.difference(encoding_vars)
-    units_vars.difference(encoding_vars)
-    description_vars.difference(encoding_vars)
-    long_name_vars.difference(encoding_vars)
-
-
-def check_sensor_configs(sensor_name: str) -> None:
-    """Check the validity of the sensor configurations.
+    data = read_config_yml(sensor_name, file_name)
+    center_len = len(data.get("center"))
+    bound_len = len(data.get("bounds"))
+    width_len = len(data.get("width"))
+
+    return [center_len, bound_len, width_len]
+
+
+def check_raw_array(sensor_name: str) -> None:
+    """Check raw array consistency from config file.
 
     Parameters
     ----------
     sensor_name : str
         Name of the sensor.
+
+    Raises
+    ------
+    ValueError
+        Error if the chuncksizes are not consistent.
     """
-    check_bin_consistency(sensor_name=sensor_name)
+    raw_data_format = read_config_yml(sensor_name, "raw_data_format.yml")
+
+    # Get keys in raw_data_format where the value is "dimension_order"
+    dict_keys_with_dimension_order = {
+        key: value.get("dimension_order") for key, value in raw_data_format.items() if "dimension_order" in value.keys()
+    }
+
+    # Get chunksizes
+    L0B_encodings = read_config_yml(sensor_name, "L0B_encodings.yml")
+
+    # Iterate over raw_data_format keys with "dimension_order" value
+    for key, list_velocity_or_diameter in dict_keys_with_dimension_order.items():
+        for i, velocity_or_diameter in enumerate(list_velocity_or_diameter):
+            # get the definiation of the chunksizes
+            chunksize_definition = L0B_encodings.get(key).get("chunksizes")[i + 1]
+
+            # define config file name
```

Comment:
Here I really don't understand what you are doing? 
Bins info is completely unrelated to chunksizes !

Reply:
Or actually, if `bin_velocity.yml` is empty... means that the disdrometer does not provide velocity measurements (see RD_80) and so `dimension_order` of RD_80 does not contain "velocity*" and `chunksizes` are max of length 2 `(time, diameter)`